docs: mention on-premise support more prominently (#2095)
berops/claudie: docs/use-cases/use-cases.md

- startups building AI/ML products
- cost-conscious organizations looking to avoid GPU pricing from hyper-scale providers

> ## 2. On-premise integration and hybrid extension

Reviewer: ```suggestion
## 2. On-premises integration and hybrid extension
```

> A company already runs (or plans to run) Kubernetes workloads on its own on-premise hardware, co-located servers, or private datacenter, and wants to keep using that investment while gaining the flexibility of public cloud. Claudie lets them register their existing machines as [static nodepools](../input-manifest/providers/on-prem.md) and combine them in a single cluster with nodepools from AWS, GCP, Azure, Hetzner, and other supported providers. The on-premise nodes are first-class members of the cluster, eligible for workloads, loadbalancing, and node-level autoscaling on the cloud side.

Reviewer: ```suggestion
A company already runs (or plans to run) Kubernetes workloads on its own on-premises hardware, co-located servers, or private datacenter, and wants to keep using that investment while gaining the flexibility of public cloud. Claudie lets them register their existing machines as [static nodepools](../input-manifest/providers/on-prem.md) and combine them in a single cluster with nodepools from AWS, GCP, Azure, Hetzner, and other supported providers. The on-premises nodes are first-class members of the cluster, eligible for workloads, loadbalancing, and node-level autoscaling on the cloud side.
```

Typical client profile:

> - organizations with significant existing on-premise or bare-metal investment

Reviewer: ```suggestion
- organizations with existing on-premises or bare-metal investment
```

> - regulated industries that must keep certain workloads in their own datacenter

Reviewer: ```suggestion
- companies in regulated industries that must keep certain workloads in their own datacenter
```

- startups
- in need of significant computing power already in their early stages (e.g. AI/ML workloads)

## 4. Cost-saving

> A company would like to utilize their s or leased resources that they already invested into, but would like to:

Author: Good catch, but the original was "on-premise or leased" and got mangled by the bulk rename. Restored to "on-premises or leased".
